### PR TITLE
Support booleans

### PR DESCRIPTION
**Pull request by simon-id**
### What does this PR do?

add a unit test for an edge case

### CODE REVIEW

File: test/rules.json [hunk 1]
```diff
@@ -1,30 +1,58 @@
...
+          "operator": "match_regex",
+          "parameters": {
+            "inputs": [
+              {
+                "address": "server.response.status"
+              }
+            ],
+            "regex": "^404$",
+            "options": {
+              "case_sensitive": true
```
Comment (simon-id) [inline test/rules.json:48]: Without this line, the test I added passes. 🤔

Comment (simon-id) [inline test/rules.json:48]: Without this line, the test I added passes. @vdeturckheim 🤔

Comment (vdeturckheim) [reply test/rules.json:48]: @Anilm3 @Taiki-San can tou confirm this is a valid option? Thanks a lot

Comment (vdeturckheim) [reply test/rules.json:48]: There is no support for booleans in bindings - fixing
